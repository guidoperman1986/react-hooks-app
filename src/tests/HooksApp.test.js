import React from 'react';
import { shallow } from "enzyme";
import { HooksApp } from '../HooksApp';

describe('Pruebas en <HookApp />', ()=>{
    test('should create', ()=>{
        const wrapper = shallow(<HooksApp />)

        expect(wrapper).toMatchSnapshot();


    })




})