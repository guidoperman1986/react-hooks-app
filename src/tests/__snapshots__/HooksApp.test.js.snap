// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HookApp /> should create 1`] = `
<div>
  Hola Mundo
</div>
`;
